src/pages/list/index.jsx

```diff
@@ -1,11 +1,73 @@
 import React from "react";
 import Header from "../../components/Header";
+import Filter from "../../components/Filter";
+import styled from "styled-components";
+import places from "../../data/places";
+
+const ListItme = styled.li`
+  padding: 18px;
+  box-sizing: border-box;
+  border-bottom: 1px solid #dbdbdb;
+`;
+
+const Image = styled.img`
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 144px;
+  object-fit: cover;
+  color: rgba(255, 255, 255, 0);
+  text-indent: -9999px;
+  border: none;
+  vertical-align: top;
+`;
+
+const Info = styled.div`
+  padding: 12px 7px 16px 0px;
+`;
+
+const Title = styled.span`
+  font-size: 1.2em;
+`;
+
+const Score = styled.span`
+  color: #ff792a;
+  font-size: 1.3em;
+  margin-left: 8px;
+`;
+
+const Address = styled.p`
+  font-size: 0.75rem;
+  margin-top: 8px;
+  color: #9b9b9b;
+`;
+
+const Review = styled.p`
+  margin-top: 10px;
+  font-size: 0.875rem;
+  line-height: 21px;
+`;
+
+function ListItems(place) {
+  return (
+    <ListItme key={place.name}>
```

Comment:
```suggestion
    <ListItem key={place.name}>
```

오타는 여전히 수정이 안되어있네요.
ListItem 을 별도의 컴포넌트로 아예 분리하고,
Container 등으로 스타일 컴포넌트를 바꾸는 것도 방법입니다.

Reply:
ListItem을 공통 컴포넌트로 만들고 Container로 감쌋습니다.

![image](https://user-images.githubusercontent.com/55589616/119927540-91703300-bfb4-11eb-9687-71c3cf55a33a.png)

![image](https://user-images.githubusercontent.com/55589616/119927558-97feaa80-bfb4-11eb-89e5-bb22faa2a980.png)